### PR TITLE
[ADD] inventory: move packaging to standalone doc

### PR DESCRIPTION
**Pull request by Felicious**
### Summary of changes
1. Move packagings doc from the [usage doc](https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/usage.html) to its own standalone doc
2. Update images to v16.3
3. Update instructions to v 16.3

### Merge Order
Merge after #7734

### FWport
All the way to master (:

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:6]:
```suggestion
In Odoo *Inventory*, *packaging* refers to disposable containers holding multiple units of a specific product. Each specific packaging must be defined on the individual product form.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:10]:
```suggestion
individual product, since packagings are product specific, and not generic.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:22]:
```suggestion
Packagings can be created directly on the product form, or from the :guilabel:`Product Packagings`
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:29]:
```suggestion
Products`, and select the desired product.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:72]:
```suggestion
   Two soda products, `Grape Soda` and `Diet Coke`, have three types of packagings configured. On the
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:73]:
```suggestion
   :guilabel:`Product Packagings` page, each product can be sold as a `6-Pack` that contains 6
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:74]:
```suggestion
   products, as a `12-Pack` of 12 products, or as a `Case` of 32 products.
```

**Review by brse-odoo — APPROVED** [review]
Hi @Felicious I've finished my review of this PR. Great job! Short, sweet. and to the point. Only had a few minor suggestions for you to accept/reject as you see fit, so I'm going to approve this for the next round.

Let me know if you have any questions! Thanks.

**Comment by Felicious** [comment]
Hi @odoo/us-doc-review , this PR is ready for you to take a look!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:6]:
```suggestion
specific product. Each specific packaging **must** be defined on the individual product form.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:10]:
```suggestion
to be configured on the individual product form. This is because packagings are product specific -
**not** generic.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:41]:
```suggestion
- :guilabel:`Packaging` (required): name of packaging that appears on sales/purchase orders as a
  packaging option for the product.
- :guilabel:`Contained quantity` (required): amount of product in the packaging.
- :guilabel:`Unit of Measure` (required): measurement unit for quantifying the product.
```
just feels like this formatting reads a bit cleaner. But, if you disagree, feel free to disregard.

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:47]:
```suggestion
   :guilabel:`(sliders)` icon to the far-right of the column titles in the :guilabel:`Packaging`
   section, and selecting the desired options from the drop-down menu that appears.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:53]:
```suggestion
- :guilabel:`Barcode`: identifier for tracing packaging in stock moves or pickings, using the
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:60]:
```suggestion
   :guilabel:`Add a line`. In the line, name the :guilabel:`Packaging` `6-pack`, and set the
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:89]:
```suggestion
:guilabel:`Packaging` field.
```
correct?

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:78]:
```suggestion
   the :guilabel:`Product Packagings` page, each product can be sold as a :guilabel:`6-Pack` that
   contains 6 products, as a :guilabel:`12-Pack` that contains 12 products, or as a :guilabel:`Case`
   that contains 32 products.
```

Comment (ksc-odoo) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:78]: used GUIs bc that's how it appears on the screenshot/UI

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:92]:
```suggestion
   18 cans of the product, :guilabel:`Grape Soda`, is packed using three :guilabel:`6-pack`
   packagings.
```

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:76]:
```suggestion
   Two soda products, :guilabel:`Grape Soda` and :guilabel:`Diet Coke`, have three types of
   packagings configured. On
```

**Review by ksc-odoo — APPROVED** [review]
Hey @Felicious great job on this doc! Just finished my Final Review, and there is only a small handful of minor adjustments that require your attention, so I'll go ahead and approve this for now. Once you make those necessary modifications/adjustments, you can feel free to tag this for Tech Review. Nice work!

File: content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/packaging.rst:79]: While it's true that we see Grape Soda on the UI,  it's an example product and doesn't appear in every customer database. Compared to something like a field or a header (like "Product Packagings"), I think grape soda is just user input and  gets back ticks `, not a guilabel (:

**Comment by Felicious** [comment]
Hi @samueljlieber! This PR is the third of a chain of stacked PRs :eyes: It's ready for a technical review from you!

For your reference: 
1st: https://github.com/odoo/documentation/pull/7659 (in final review)
2nd: https://github.com/odoo/documentation/pull/7734 (in final review)
3rd: this one!

**Comment by StraubCreative** [comment]
Holding off on review until #7734 is merged. Will need to rebase and change base branch.

**Comment by Felicious** [comment]
#7734 is merged, and rebased this PR to 16.0!

**Comment by StraubCreative** [comment]
- #7734 + fwps are all merged
- Rebased to latest on [6bed4df](https://github.com/odoo/documentation/pull/7751/commits/6bed4df88dc5b59a46c87fe3e5f99803707e6b39)
- All checks pass ✅ 
...
@robodoo r+

**Pull request closed without merging**
